### PR TITLE
Modified index

### PR DESCRIPTION
**Pull request by hsorellana**
Hey everyone!

Made a few, and little, changes on the index due to some missing title indexing. All started when i clicked on the "slide" entry and it didn't do anything. Of course, now that entry is gone haha.

If i missed something, let me know to fix it 😄

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,18 +25,20 @@ All the links are monitored and tested with [awesome_bot](https://github.com/dkh
...
 	- [Dockerfile](#dockerfile)
 	- [Garbage Collection](#garbage-collection)
 	- [Hosting Images (registries)](#hosting-images-registries)
-	- [Image Builder](#image-builder)
 	- [Images](#images)
+	- [Image Builder](#image-builder)
 	- [Linter / Validator](#linter--validator)
 	- [Local Container Manager](#local-container-manager)
 	- [Monitoring & Logging](#monitoring--logging)
+		- [Monitoring & Logging Services](#monitoring-&-logging-services)
```
Comment (veggiemonk) [inline README.md]: This one does not work.

File: README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 # Awesome Docker [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome) [![Join the chat at https://gitter.im/veggiemonk/awesome-docker](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/veggiemonk/awesome-docker?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) [![Say Thanks](https://img.shields.io/badge/SayThanks.io-%E2%98%BC-1EAEDB.svg)](https://saythanks.io/to/veggiemonk)
 
 > A curated list of Docker resources and projects
-Inspired by [@sindresorhus](https://github.com/sindresorhus)' [awesome][sindresorhus] and improved by these **[amazing contributors](https://github.com/veggiemonk/awesome-docker/graphs/contributors)**.
+Inspired by [@sindresorhus](https://github.com/sindresorhus)'s [awesome][sindresorhus] and improved by these **[amazing contributors](https://github.com/veggiemonk/awesome-docker/graphs/contributors)**.
```
Comment (veggiemonk) [inline README.md]: Are you sure about this?

File: README.md [hunk 3]
```diff
@@ -50,15 +52,21 @@ All the links are monitored and tested with [awesome_bot](https://github.com/dkh
 	- [Utilities](#utilities)
 	- [Volume management and plugins](#volume-management-and-plugins)
 	- [Web Interface](#web-interface)
-- [Slides](#slides)
 - [Useful Resources](#useful-resources)
+	- [Good Tips](#good-tips)
```
Comment (veggiemonk) [inline README.md]: Nice! 👍

File: README.md (hunk 2 above)
Comment (devhkr) [inline README.md]: I agree with @veggiemonk, the 's' can be omitted.

**Comment by devhkr** [comment]
I was also about to fix the index. But thanks @hsorellana 👍

File: README.md [hunk 4]
```diff
@@ -50,15 +52,21 @@ All the links are monitored and tested with [awesome_bot](https://github.com/dkh
...
+	- [Raspberry Pi / ARM](#raspberry-pi-&-arm)
 - [Videos](#videos)
 	- [Main Account](#main-account)
 	- [Useful videos](#useful-videos)
-- [Interactive Learning Environments](#interactive-learning-environments)
 - [Interesting Twitter Accounts](#interesting-twitter-accounts)
 	- [People](#people)
 - [Communities and Meetups](#communities-and-meetups)
+	- [Brazilian](#brazilian)
+	- [Chinise](#chinise)
```
Comment (devhkr) [inline README.md]: Because of a small typo, this linking doesn't work either.

File: README.md (hunk 1 above)
Comment (devhkr) [inline README.md]: @hsorellana I found more of these, you should delete all the '&' symbols in the anchor tags to make linking work. So,
```markdown
[Monitoring & Logging Services](#monitoring-&-logging-services)
```
becomes
```markdown
[Monitoring & Logging Services](#monitoring--logging-services)
```

File: README.md (hunk 2 above)
Comment (hsorellana) [inline README.md]: i'll erase it then ;)

File: README.md [hunk 5]
```diff
@@ -50,15 +52,21 @@ All the links are monitored and tested with [awesome_bot](https://github.com/dkh
...
+	- [Good Tips](#good-tips)
+	- [Newsletter](#newsletter)
+	- [Security](#security)
+	- [Raspberry Pi / ARM](#raspberry-pi-&-arm)
 - [Videos](#videos)
 	- [Main Account](#main-account)
 	- [Useful videos](#useful-videos)
-- [Interactive Learning Environments](#interactive-learning-environments)
 - [Interesting Twitter Accounts](#interesting-twitter-accounts)
 	- [People](#people)
```
Comment (devhkr) [inline README.md]:
```markdown
- [People](#people)
```
can be deleted since the title is gone.

Comment (hsorellana) [inline README.md]: nice! thanks!

File: README.md [hunk 6]
```diff
@@ -50,15 +52,20 @@ All the links are monitored and tested with [awesome_bot](https://github.com/dkh
...
+	- [Raspberry Pi / ARM](#raspberry-pi--arm)
 - [Videos](#videos)
 	- [Main Account](#main-account)
 	- [Useful videos](#useful-videos)
-- [Interactive Learning Environments](#interactive-learning-environments)
 - [Interesting Twitter Accounts](#interesting-twitter-accounts)
-	- [People](#people)
 - [Communities and Meetups](#communities-and-meetups)
+	- [Brazilian](#brazilian)
+	- [Chinise](#chinise)
```
Comment (devhkr) [inline README.md]:
```markdown
 +	- [Chinise](#chinise)
```
small typo here. Change it to Chinese/chinese.

Comment (hsorellana) [inline README.md]: that's why i didn't like how that word looked but i never saw the mistake haha

**Pull request merged**